src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double width;
+    private double height;
+
+    public Rectangle(Color color, double width, double height) {
+        super(color);
+        this.width = width;
+        this.height = height;
+    }
+
+    @Override
+    public double calculateArea() {
+        return Math.round(width * height);
```

Comment:
The `calculateArea` method rounds the area, which might not be necessary. Consider returning the precise value of the area without rounding, as rounding can lead to loss of precision.